### DISCUSSION TITLE
Official docker image

### DISCUSSION THREAD

**Discussion by Zerotask** [microsoft/garnet#101] [Ideas]
Would be great to have an official docker image

**Comment by douglasg14b** [comment]
Looks like this may have been completed here: https://github.com/microsoft/garnet/pull/107

You can find the images here: https://github.com/microsoft/garnet/pkgs/container/garnet

However, docker-image specific documentation alongside the container registry image would be nice.

**Comment by badrishc** [comment]
https://microsoft.github.io/garnet/docs/welcome/releases#docker
